### ISSUE TITLE
Needs Author Record and link in BHSE? Michael the Companion of Angels

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#288]
Nathan, could you please mark this record up as an author and perhaps add links in BHSE if possible?
<tei:div xmlns:tei="http://www.tei-c.org/ns/1.0">
        <idno xmlns="http://www.tei-c.org/ns/1.0" type="URI">http://syriaca.org/person/1376/tei</idno>
        <persName xmlns="http://www.tei-c.org/ns/1.0" xml:id="name1376-2" xml:lang="en">Michael the Companion of Angels</persName>
        <persName xmlns="http://www.tei-c.org/ns/1.0" xml:id="name1376-3" xml:lang="en">Michael</persName>
        <note xmlns="http://www.tei-c.org/ns/1.0" type="abstract">In hagiography: Michael the Companion of Angels was a monk linked to <persName ref="http://syriaca.org/person/356">Mar Awgin</persName>. In some traditions, he was the
          hagiographer of the Life of Awgin. He was commemorated with other monastic founders in
          <placeName ref="http://syriaca.org/place/1452">Mesopotamia</placeName>.</note>
      </tei:div>

**Comment by wlpotter** [comment]
I don't think this was resolved? The record (https://syriaca.org/person/1376) is not marked as an author and doesn't appear to have author links, though http://syriaca.org/work/1303 is there as a commemoration work.
